gammapy/cube/fit.py

```diff
@@ -267,6 +268,63 @@ def npred(self):
 
         return npred_total
 
+    @classmethod
+    def create(cls, geom, geom_irf=None, migra_axis=None, rad_axis=None):
```

Comment:
Note tha we also need a `reference_time` to create the empty `GTI` table see e.g. https://github.com/gammapy/gammapy/blob/master/gammapy/spectrum/dataset.py#L370.

Reply:
If I make it with empty start and stop times instead of zeros, is the reference time still important?

Reply:
Yes, it is part of the meta data of the GTI table, that's why we define it in the beginning. Alternatively one could probably define a logic to take the reference time from the first GTI, that is stacked into the table, but this seem much more complicated...